src/lib.rs

```diff
@@ -1022,7 +1021,7 @@ mod test {
         // check read, write
         let mut new_se = SelfEncryptor::new(my_storage.clone(), data_map);
         let fetched = new_se.read(0, MIN_CHUNK_SIZE as u64 * 3);
-        assert_eq!(fetched, the_bytes);
```

Comment:
Because when assert_eq! fails it will print the left and right args so in this case the contents of the vectors - that's quite a screenful.